src/test/scala/is/hail/expr/ir/AggregatorsSuite.scala

```diff
@@ -120,4 +121,88 @@ class AggregatorsSuite {
       (FastIndexedSeq(Row(1.0, 10.0), Row(10.0, 10.0), Row(null, 10.0)), TStruct("a" -> TFloat64(), "b" -> TFloat64())),
       110.0)
   }
+
+  private[this] def assertArraySumEvalsTo[T: HailRep](
+    a: IndexedSeq[Seq[T]],
+    expected: Seq[T]
+  ): Unit = {
+    val aggSig = AggSignature(Sum(), TArray(hailType[T]), FastSeq(), None)
+    val aggregable = a.map(Row(_))
+    assertEvalsTo(
+      ApplyAggOp(SeqOp(Ref("a", TArray(hailType[T])), I32(0), aggSig), FastSeq(), None, aggSig),
+      (aggregable, TStruct("a" -> TArray(hailType[T]))),
+      expected)
```

Comment:
133 and 134 are two different arguments to `assertEvalsTo`, it seems to read better to me with them on separate lines because it's more clear that there are three arguments to `assertEvalsTo`